omnibus/config/software/elasticsearch.rb

```diff
@@ -35,6 +35,11 @@
          sha512: "bb02a5dc1caef97638a959ebba05dd649083c856334f30c670b851067292d7230e561d8759b15a80be73537d7a7efd9cef427d253cbb2efdbd6b168c6f9baa13"
 end
 
+version "7.9.1" do
```

Comment:
We are not changing the default version of es with this change, so none of these change will work for internal es configuration.